packages/explorer/src/app/(explorer)/[chainName]/worlds/WorldsForm.tsx

```diff
@@ -25,6 +26,9 @@ const formSchema = z.object({
 });
 
 export function WorldsForm({ worlds }: { worlds: Address[] }) {
+  // Initialize the observer store to start fetching transactions
+  store;
```

Comment:
```suggestion
  // Reference to imported store observer so the listener is established when this component is used.
  store;
```
also wonder if we could do the same thing outside this component or if it has to be referenced inside to be considered in scope of the page bundle

Reply:
also wondering about doing this inside the top-level `layout.tsx` instead